### ISSUE TITLE
Can not install module on 2.4.6

### ISSUE THREAD

**Issue by JelleGe** [yireo/Yireo_LinkPreload#42]
When we try to install the latest version on 2.4.6, we fetch the following error:

```
  Problem 1
    - yireo/magento2-linkpreload[1.4.0, ..., 1.4.4] require symfony/css-selector ^2.7|^3.0|^4.0 -> found symfony/css-selector[v2.7.0, ..., v2.8.52, v3.0.0, ..., v3.4.47, v4.0.0, ..., v4.4.44] but the package is fixed to v6.2.7 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - yireo/magento2-linkpreload[1.4.5, ..., 1.4.19] require symfony/css-selector ^2.7|^3.0|^4.0|^5.0 -> found symfony/css-selector[v2.7.0, ..., v2.8.52, v3.0.0, ..., v3.4.47, v4.0.0, ..., v4.4.44, v5.0.0, ..., v5.4.21] but the package is fixed to v6.2.7 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - Root composer.json requires yireo/magento2-linkpreload ^1.4 -> satisfiable by yireo/magento2-linkpreload[1.4.0, ..., 1.4.19].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
You can also try re-running composer require with an explicit version constraint, e.g. "composer require yireo/magento2-linkpreload:*" to figure out if any version is installable, or "composer require yireo/magento2-linkpreload:^2.1" if you know
```

**Comment by jissereitsma** [comment]
Thanks. There is actually no issue with Symfony 6 at all, so I have released a new version 1.4.20 to fix these compatibility issues.

**Comment by jissereitsma** [comment]
I assume this issue is solved.
